src/TMedia.h

```diff
@@ -130,8 +130,8 @@ class TMedia : public QObject
 
 private:
     QUrl parseUrl(TMediaData& mediaData);
-    bool isValidUrl(QUrl& url);
-    bool isFileRelative(TMediaData& mediaData);
+    static bool isValidUrl(QUrl& url);
```

Comment:
I am not sure I get the reasons behind this type of change, would you care to elaborate Vadi?

Reply:
They don't access members, thus can be made static - helps readability: https://clang.llvm.org/extra/clang-tidy/checks/readability-convert-member-functions-to-static.html

Reply:
:man_facepalming: Oh, yeah, of course - ~~and can be accessed outside of the class as `TMedia::isValidUrl(Qrl&)`~~ no it can't because it is private. :stuck_out_tongue_closed_eyes: